.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Don't request an update of a package,
+        [We have a script for that](https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt).
+        However, a quality pull request may help.
+  - id: xuname
+    type: input
+    attributes:
+      label: System Info
+      description: Output of `xuname` (part of [`xtools`](https://man.voidlinux.org/xtools.1))
+      placeholder: Void 5.x.y_z x86_64-musl ...
+    validations:
+      required: true
+  - id: packages
+    type: input
+    attributes:
+      label: Package(s) Affected
+      description: Affected package(s) including version (this can be found with `xbps-query -p pkgver foo`)
+      placeholder: foo-1.0.2_5, bar-5.6.7_1, baz-0.0.3_5, ...
+    validations:
+      required: true
+  - id: upstream
+    type: textarea
+    attributes:
+      label: Does a report exist for this bug with the project's home (upstream) and/or another distro?
+      description: If so, link it here (It's fine if there's none)
+      placeholder: |
+        For example:
+        https://bugs.kde.org/show_bug.cgi?id=432975
+        https://bugs.gentoo.org/767478
+  - id: expected
+    type: textarea
+    attributes:
+      label: Expected behaviour
+      description: A clear and concise description of what you expected to happen
+      placeholder: The package is supposed to do this thing.
+    validations:
+      required: true
+  - id: description
+    type: textarea
+    attributes:
+      label: Actual behaviour
+      description: A clear and concise description of what the bug is
+      placeholder: There was a crash when...
+    validations:
+      required: true
+  - id: steps
+    type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Clear steps to reproduce the bug
+      placeholder: |
+        1. Do the thing
+        2. Do the other thing
+        3. ???
+        4. Crash :(
```

Comment:
![image](https://user-images.githubusercontent.com/23639164/163478507-f2e63719-96f2-4515-ab51-9403a4bbed2b.png)
Newlines are missing in a webkit based browser, but I don't think we can fix that.

Reply:
works fine in chromium-based, but yeah I think you're right

Reply:
This is from Epiphany 42 (GNOME Web).

Reply:
hm, sounds like something that should be reported [upstream](https://github.com/github/feedback/discussions/categories/issues-feedback)